python_testspace_xml/testspace_xml.py

```diff
@@ -184,7 +185,7 @@ def add_annotation(self, annotation):
 class TestSuite:
     def __init__(self, name):
         # optional sub-suites
-        self.sub_suites = {}
+        self.sub_suites = collections.OrderedDict()
```

Comment:
as we discussed, and considering that it is common the same Suite to be run multiple time and have result in duplications, it should better be a `list`